Resources/public/js/views/ez-languageselectionboxview.js

```diff
@@ -344,6 +361,14 @@ YUI.add('ez-languageselectionboxview', function (Y) {
             },
 
             /**
+             * List of active system languages. List contains language objects.
```

Comment:
You should mentioned somewhere that the language objects are indexed by language code.